packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -23,6 +24,16 @@ export interface PipelineProps {
    * and uses that for the pipeline name.
    */
   pipelineName?: string;
+
+  /**
+   * A map of region to S3 Bucket used for cross-region CodePipeline.
+   * For every Action that you specify living in a different region than the Pipeline itself,
+   * if you don't provide an explicit Bucket for that region using this property,
+   * the construct will automatically create a scaffold Stack containing an S3 Bucket in that region.
+   * Note that you will have to `cdk deploy` that Stack before you can deploy your Pipeline-containing Stack.
+   * You can query the generated Stacks using the {@link Pipeline#crossRegionScaffoldStacks} property.
```

Comment:
Can we get a doc writer from CodePipeline to help phrase these inline with the official documentation? I feel we need to align with it better.

Reply:
The paragraph you highlighted is 100% about the CDK, so I'm not sure if a CodePipeline writer would help. Maybe we should ask Doug? Anyway, I think this should be done as a follow-up, not necessarily in this PR.